docs/admin/access.rst

```diff
@@ -1,333 +1,433 @@
+        "User" -- "Group";
+        "Group" -- "Role";
+        "Role" -- "Permission";
+        "Group" -- "Project";
+        "Group" -- "Language";
+        "Group" -- "Components";
+        "Group" -- "Component list";
+    }
 
-.. describe:: Sources
+.. note::
 
-    Can edit source strings in :ref:`monolingual` and source string info.
+  A group can have no roles assigned to it, in that case access to browse the
+  project is assumed (see bellow).
 
-.. describe:: Languages
+Access for browse to a project
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    Can manage translated languages (add or remove translations).
+A user has to be a member of a group linked to the project, or any component
+inside that project. Having membership is enough, no specific permissions are
+needed to browse the project (this is used in the default `Viewers` group, see
+:ref:`default-groups`).
 
-.. describe:: Glossary
+Access for browse to a component
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-    Can manage glossary (add or remove entries, or upload).
+A user can access unrestricted components once able to access the components'
+project (and will have all the permissions he or she was granted for the
+project). With :ref:`component-restricted` turned on, access to the component
+requires explicit permissions for the component (or a component list it is in).
 
-.. describe:: Memory
+.. _perm-check:
 
-    Can manage translation memory.
+Scope of groups
+^^^^^^^^^^^^^^^
 
-.. describe:: Screenshots
+The scope of the permission assigned by the roles in the groups are applied by
+the following rules:
 
-    Can manage screenshots (add or remove them, and associate them to source
-    strings).
+- If the group specifies any :guilabel:`Component list` all the permissions given to
+  the members of that group are granted for all the components in the component
+  lists attached to the group and an access with no additional permissions is
+  granted for all the projects these components are in. :guilabel:`Components`
+  and :guilabel:`Projects` are ignored.
 
-.. describe:: Review
+- If the group specifies any :guilabel:`Components` all the permissions given to
+  the members of that group are granted for all the components attached to the
+  group and an access with no additional permissions is granted for all the
+  projects these components are in. :guilabel:`Projects` are ignored.
 
-    Can approve translations during review.
+- Otherwise if the group specifies any :guilabel:`Projects` either by directly
+  listing them or by :guilabel:`Selected projects` set to a value like `All
+  public`, all the permissions are applied to all the projects, which
+  effectively grants the same permissions to access all the projects'
+  :ref:`unrestricted components <component-restricted>`.
 
-.. describe:: VCS
+- The restrictions imposed by group's :guilabel:`Languages` are applied later,
+  when the access to perform certain action is checked, and it only affects just
```

Comment:
```suggestion
  when the access to perform certain action is checked, and it only affects just
```
Can we avoid "checked" as in also checkbox-checked?

Reply:
done

Reply:
It's should be `verified`, not `enabled`.

Reply:
fixed that